sfxclient/sfxclient.go

```diff
@@ -290,6 +291,11 @@ func (s *Scheduler) ReportOnce(ctx context.Context) error {
 		s.previousDatapoints = datapoints
 		return datapoints
 	}()
+	if s.Prefix != "" {
```

Comment:
You'll also need to check s.Prefix != nil

Reply:
It is a string so init value is ""

Reply:
Ah yeah, I was thinking that it was a *string